packages/eas-cli/src/commandUtils/EasCommand.ts

```diff
@@ -21,13 +21,16 @@ export default abstract class EasCommand extends Command {
    * force the user to log in
    */
   protected requiresAuthentication = true;
+  protected mustBeRunInsideAnExpoProject = true;
```

Comment:
```suggestion
  protected mustBeRunInsideProject = true;
```
It doesn't have to be an expo project. With EAS Build, you can build any React Native project.